.github/workflows/review.yml

```diff
@@ -99,9 +99,8 @@ jobs:
       fetch_cmd_aarch64-darwin: ${{ steps.report.outputs.fetch_cmd_aarch64-darwin }}
 
     steps:
-      - name: prepare /nix
-        run: sudo mkdir /mnt/nix && sudo mount -m -o bind /mnt/nix /nix
-        if: ${{ matrix.system == 'x86_64-linux' || matrix.system == 'aarch64-linux' }}
+      - uses: wimpysworld/nothing-but-nix@26234b059dd22b36374427bd3cea6adfe16a809f
+        if: ${{ runner.os == 'Linux' }}
```

Comment:
Should this use one of the workarounds available for https://github.com/wimpysworld/nothing-but-nix/issues/18 ?

E.g:

```yaml
  with:
    hatchet-protocol: 'holster'
```

or

```yaml
- name: Create TMPDIR
  if: ${{ runner.os == 'Linux' }}
  run: |
    mkdir -p /nix/_tmp
    echo "TMPDIR=/nix/_tmp" >> "$GITHUB_ENV"
```

Reply:
Hasn't this already been solved upstream? See https://github.com/Defelo/nixpkgs-review-gha/actions/runs/16028458182/job/45222455424#step:2:190

> TMPDIR set to /mnt/tmp.pv22VZtzz2 to use space on the /mnt volume for builds.

Reply:
> The other approach we could take that upstream can't, is explicitly setting `build-dir = /nix/_build` in `extra_nix_config`. We'd just need to test whether nix will create the directory if it doesn't exist.

Tested this, and it seems we _do_ need to ensure that `build-dir` exists prior to building anything.

https://github.com/MattSturgeon/nixpkgs-review-gha/actions/runs/16031964511/job/45234659527#step:7:702